tests/test_simulator.py

```diff
@@ -44,6 +47,7 @@ def clean():
     return
 
 
+@pytest.mark.skipif(get_docker_client().get_os_type().lower() == 'windows', reason='Simulator does not support windows container')
```

Comment:
These `get_docker_client().get_os_type().lower()` code can be extracted to a method.

Reply:
Updated code.